### ISSUE TITLE
pacreport: `IgnoreMissing` option

### ISSUE THREAD

**Issue by mizzunet** [andrewgregory/pacutils#56]
Like
```
IgnoreMissing =  /usr/share/locale/*
```

As I have deleted unwanted locale files

**Comment by andrewgregory** [comment]
I don't think it makes sense to add an option for this.  I'm thinking pacreport should either just ignore anything in NoExtract or maybe the missing file check should just be removed in favor of paccheck.
